src/wallet/generateFaucetWallet.ts

```diff
@@ -5,6 +5,7 @@ import { errors } from "../common";
 import { RippledError } from "../common/errors";
 import { isValidAddress } from "../common/schema-validator";
 import { GeneratedAddress } from "../utils/generateAddress";
+import Wallet from ".";
```

Comment:
This likely causes a dependency cycle

Reply:
From my understanding, a dependency cycle would occur if `wallet/index.ts` and `generateFaucetWallet.ts `referenced each other which isn't the case. Only `generateFaucetWallet.ts` references `wallet/index.ts`.